doc/conf.py

```diff
@@ -497,6 +505,7 @@ def notebook_modification_function(notebook_content, notebook_filename):
     "inspect_global_variables": False,
     "remove_config_comments": True,
     "plot_gallery": "True",
+    "reset_modules": ("matplotlib", "seaborn", reset_sklearn_config),
```

Comment:
question: why can't we simply say `sklearn` here? Also, shouldn't `numpy` (and other deps maybe) be here too? One could imagine an example where numpy's print options are modified for example.

Reply:
The "matplotlib" and "seaborn" are sphinx-gallery built-in reset functions. They are not implemented for every module.